x/crosschain/keeper/msg_server_add_outbound_tracker.go

```diff
@@ -63,6 +63,12 @@ func (k msgServer) AddOutboundTracker(
 		return nil, cosmoserrors.Wrapf(authoritytypes.ErrUnauthorized, "Creator %s", msg.Creator)
 	}
 
+	// set the outbound hash from the last tracker and save it in the store
+	// this value is helpful for explorer or front-end application to find the outbound hash, it has no on-chain utility
+	// the hash will be replaced with the actual hash, if different, when the outbound transaction is observed and voted
+	cctx.CrossChainTx.GetCurrentOutboundParam().Hash = msg.TxHash
+	k.SetCrossChainTx(ctx, *cctx.CrossChainTx)
+
```

Comment:
On relayer style chains like solana doing it like this will result in a failed hash being displayed quite often? Since there are N relayers submitting the transaction at the same time?

Reply:
There will be no issue if a false hash is acceptable for intended use cases (e.g., if explorers and front-end apps can verify the hash by themselves).

Reply:
Yes, this can be managed in the front-end side